source/js/widgets/drag-view.js

```diff
@@ -122,6 +122,15 @@ export class DragView extends HTMLElement {
           left: e.clientX - framePosition.x - 10 + 'px',
           top: e.clientY - framePosition.y - 20 + 'px'
         }
+        if (this.shadowRoot.querySelector('.lastselected')) {
```

Comment:
There is a logic to remember the last focused text and the new lastselected logic is duplicate and it breaks the function of  inheriting the color and fontsize by the subsequent textboxes.

Reply:
fixed